services/product-service/app/config.py

```
@@ -0,0 +1,48 @@
+
+
+class Settings(BaseSettings):
+    model_config = SettingsConfigDict(env_file=env_path, extra="ignore", env_file_encoding="utf-8")
+
+    # 서비스 식별
+    service_name: str = "product-service"
+    service_version: str = "0.1.0"
+    debug: bool = False
+
+    # 내부 서비스 토큰
+    internal_service_token: str = "change-me-in-production"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "internal_service_token"

(S105)